doc/release_notes_1.0.rst

```diff
@@ -0,0 +1,478 @@
+more information.)
+
+Device pass-through support
+===========================
+Device pass-through to UOS supported.
+OVMF support initial patches merged in ACRN: To support booting Windows as a 
+Guest OS, we are using Open source Virtual Machine Firmware (OVMF). Initial 
+patches to support OVMF have been merged in ACRN hypervisor. More patches for 
+ACRN and patches upstreaming to OVMF work will be continuing.
+
+Support ACPI power key mediator
+===============================
+ACRN supports ACPI power/sleep key on the APL and KBL NUC platforms, triggering 
+S3/S5 flow, following the ACPI spec.
+
+GPIO virtualization
+===================
+GPIO virtualization is supported as para-virtualization based on the Virtual 
+I/O Device (VIRTIO) specification. The GPIO consumers of the Front-end are able
+to set or get GPIO values, directions, and configuration via one virtual GPIO 
+controller. In the Back-end, the GPIO command line in the launch script can be 
+modified to map native GPIO to UOS.
+
+New ACRN tools
+==============
+We’ve added a collection of support tools including acrnctl, acrntrace, acrnlog, 
+acrn-crashlog, acrnprobe. (See Tools documentation for details.)
+
+Document updates
+================
+Several new documents have been published in v1.0 release, including:
+
+* :ref:`GPU Passthrough on Skylake NUC <skl-nuc-gpu-passthrough>`
+* :ref:`Device Model Parameters <acrn-dm_parameters>`
+* :ref:`Running Automotive Grade Linux as a VM <agl-vms>`
+* :ref:`Using PREEMPT_RT-Linux for real-time UOS <rt_linux_setup>`
+* :ref:`Frequently Asked Questions <faq>`
+* :ref:`An introduction to Trusty and Security services on ACRN <trusty-security-services>`
+* A Wiki article about `Porting ClearLinux/ACRN to support Yocto/ACRN
+  <https://github.com/projectacrn/acrn-hypervisor/wiki/Yocto-based-Service-OS-(SOS)-and-User-OS-(UOS)-on-ACRN>`_
+* An `ACRN brochure update (English and Chinese)
+  <https://projectacrn.org/#code-docs>`_
+* ACRN Roadmap: look ahead in `2019
+  <https://projectacrn.org/wp-content/uploads/sites/59/2019/02/ACRN-Rodamap-2019.pdf>`_
+* Performance analysis of `VBS-k framework
+  <https://projectacrn.github.io/latest/developer-guides/VBSK-analysis.html>`_
+* HLD design doc for `IOC virtualization
+  <https://projectacrn.github.io/latest/developer-guides/hld/hld-APL_GVT-g.html?highlight=hld>`_
+* Additional project `coding guidelines
+  <coding_guidelines.html>`_
+* :ref:`Zephyr RTOS as Guest OS <using_zephyr_as_uos>`
+* :ref:`Enable cache QoS with CAT <using_cat_up2>`
+* :ref:`ACRN kernel parameter introduction <kernel-parameters>`
+* :ref:`FAQ update for two issues <faq>`
+* :ref:`ACRN Debug introduction <acrn-debug>`
+* USB controller (including USB2.0 and 3.0 disk, keyboard, mouse)
+* SD card (mount, read, and write directly in the UOS)
+* Converged Security Engine (`CSE
+  <https://projectacrn.github.io/latest/developer-guides/hld/hld-security.html?highlight=CSE>`_)
+
+New Features Details
```

Comment:
Are these ALL the features added since 0.1?  You list lots of documents in the previous section, but not many features here and there were over 2200 PRs closed.